src/chainparams.cpp

```diff
@@ -241,6 +243,8 @@ class CTestNetParams : public CChainParams {
         consensus.nPowMaxAdjustDown = 32; // 32% adjustment down
         consensus.nPowMaxAdjustUp = 16; // 16% adjustment up
         consensus.nPowTargetSpacing = 2.5 * 60;
+        consensus.vUpgrades[Consensus::UPGRADE_TESTDUMMY].nActivationHeight = 100;
```

Comment:
I'm not sure about leaving this here... With the rewind logic, this will cause everyone's testnet chains to be unnecessarily re-orged (trivially - the old chain will be re-connected). But I also want to have a test case with an activation height... I'll look into adding a testnet-only (or maybe regtest-only) internal API for setting an activation height.